skimage/measure/simple_metrics.py

```diff
@@ -142,3 +147,51 @@ def compare_psnr(im_true, im_test, data_range=None):
 
     err = compare_mse(im_true, im_test)
     return 10 * np.log10((data_range ** 2) / err)
+
+
+def enhancement_measure(image: np.ndarray,
+                        eps: float = 1e-7,
+                        size: int = 3) -> float:
+    """
+        The image enhancement measure called EME based on [1]_.
+
+        Parameters
+        ----------
+        image : array
+            Input image of which the quality should be assessed.
+            Can be either 3-channel RGB or 1-channel grayscale.
+            The function converts pixel intensities into floats by default.
+        size : int
+            Size of the window. Default value is 3.
+        eps : float
+            Parameter to avoid division by zero. Default value is 1e-7.
+        Returns
+        -------
+        eme : float
+            The number describing image quality.
+
+        References
+        ----------
+        .. [1] Agaian, Sos S., Karen Panetta, and Artyom M. Grigoryan.
+               "A new measure of image enhancement."
+               IASTED International Conference on Signal Processing
+               & Communication. Citeseer, 2000.
+        Examples
+        --------
+        >>> from skimage.data import camera
+        >>> from skimage.exposure import equalize_hist
+        >>> img = camera()
+        >>> quality(img) # before
+            0.9096745071475523
+        >>> quality(equalize_hist(img))
+            1.299327371881219
+
+    """
+
+    image = img_as_float(image)
+    image = rescale_intensity(image, out_range=(0., 1.))
+    eme = np.zeros_like(image)
```

Comment:
I think this line is not needed.

Reply:
> Is there a reason to rescale between [0, 1] in particular? If we were to chose a value above zero as the lower bound we could omit the `eps` parameter altogether.

Thanks for the feedback, I will get to all the stuff mentioned above as soon as I can.
Meanwhile, regarding rescaling, when I was testing images with range `[-1,1]` I ran into issues with logarithms, since some pixel neighborhoods can have multiple negative values, the logarithm would try to evaluate negatives and return `nan`. Hence the rescaling to `[0,1]`.